fpdf/fpdf.py

```diff
@@ -3511,22 +3512,23 @@ def multi_cell(
             clearance_margins.append(self.c_margin)
         if not padding.right:
             clearance_margins.append(self.c_margin)
-        self.x += padding.left
+        if align != Align.X:
```

Comment:
Nice catch! 👍 

I see you have added tests for all alignment options together with padding, which we apparently didn't have before.